test/e2e/singlecluster/pod_test.go

```diff
@@ -50,9 +46,6 @@ var _ = ginkgo.Describe("Pod groups", func() {
 	)
 
 	ginkgo.BeforeEach(func() {
-		if kubeVersion().LessThan(kubeversion.KubeVersion1_27) {
```

Comment:
Do we need `kubeversion.KubeVersion1_27` variable?

Reply:
I see, technically 1.27 passed EOL (end of life), but I think some users may still be on it.
I'm ok to keep it for 0.9, and we consider dropping it for 0.10. WDYT @tenzen-y ?

Reply:
As I can see, the major cloud vendors including GKE, they already stopped the v1.27 supporting in not LTS channel. So, I'm ok with removal of 1.27 from here.

But, Openshift still provides maintenance support until a half year later. Additionally, this 1.27 dedicated testing mechanism does not significantly affect problems in our integration testing mechanism. So, I agree with @mimowo idea.

https://endoflife.date/red-hat-openshift

(4.14 == 1.27)

Reply:
Dear users who still use 1.27, note that 1.27 is an implied supported version, and it is not a publicly supported version. So, we do not have any plan to provide bug and regression fixes for the version.